spec/lib/pubmed/map_pub_hash_spec.rb

```diff
@@ -0,0 +1,237 @@
+# frozen_string_literal: true
```

Comment:
this gets the pub hash related specs from the source record ported over, along with some refactoring